packages/psss-server/src/routes/alerts/FetchAlertsRoute.ts

```diff
@@ -64,7 +65,7 @@ export class FetchAlertsRoute extends Route {
     const { results } = await this.reportConnection.fetchReport(
       `PSSS_${syndrome}_Confirmed_Report`,
       [organisationUnit],
-      [period],
+      [period, getCurrentPeriod('WEEK')],
```

Comment:
To make `Cases since alert began` cumulative til today. I think it makes more sense to always sum all the cases til today instead of to the selected week

Reply:
Good point!